pkg/v1/model/viewwidget.go

```diff
@@ -0,0 +1,20 @@
+package model
+
+import "gorm.io/datatypes"
+
+type ViewWidget struct {
+	CommonUUID
+	Name        string         `json:"name" gorm:"type:varchar(255);not null;uniqueIndex:idx_view_widgets_name_view_uuid;"`
+	Order       int            `json:"order"`
+	X           int            `json:"x"`
+	Y           int            `json:"y"`
+	ViewUUID    string         `json:"view_uuid,omitempty" gorm:"type:varchar(255) references views;uniqueIndex:idx_view_widgets_name_view_uuid"`
+	Type        *string        `json:"type"`
+	Config      datatypes.JSON `json:"config"`
+	NetworkName string         `json:"network_name"`
+	DeviceName  string         `json:"device_name"`
+	PointName   string         `json:"point_name"`
+	HostUUID    string         `json:"host_uuid,omitempty" gorm:"uniqueIndex:idx_view_widgets_host_uuid_point_uuid"`
```

Comment:
`uniqueIndex: idx_view_widgets_host_uuid_point_uuid` should not be there. The same view should be able to point to the same point multiple times in the widget. Let's say, the same point with value display, slider, switch, and on and on...